assets/section-main-product.css

```diff
@@ -1178,3 +1195,65 @@ a.product__text {
   display: block;
   padding-bottom: 100%;
 }
+
+/* collapsed grid overrides */
+@media screen and (min-width: 750px) {
+  .product--stacked .grid--collapsed-y .grid__item:first-child {
+    --top-left-radius: var(--border-radius);
+    --top-right-radius: var(--border-radius);
+  }
+
+  .product--thumbnail .grid--collapsed .is-active,
+  .product--thumbnail_slider .grid--collapsed .is-active {
+    --top-left-radius: var(--border-radius);
+    --top-right-radius: var(--border-radius);
+    --bottom-right-radius: var(--border-radius);
+    --bottom-left-radius: var(--border-radius);
+  }
+}
+
+@media screen and (min-width: 750px) and (max-width: 989px) {
+  .product--stacked .grid--collapsed-y .grid__item:last-child,
+  .product--stacked .grid--collapsed-x:not(.grid--collapsed-y) .grid__item {
+    --bottom-right-radius: var(--border-radius);
+    --bottom-left-radius: var(--border-radius);
+  }
+
+  .product--stacked .grid--collapsed-x:not(.grid--collapsed-y) .grid__item {
+    --top-left-radius: var(--border-radius);
+    --top-right-radius: var(--border-radius);
+  }
+}
+
+@media screen and (min-width: 990px) {
+  .product--stacked .grid--collapsed-y .grid__item:first-child {
+    --top-left-radius: var(--border-radius);
+    --top-right-radius: var(--border-radius);
+  }
+
+  .product--stacked .grid--collapsed-y .grid__item:only-child,
+  .product--stacked .grid--collapsed-y .grid__item:last-child:nth-child(even) {
+    --bottom-left-radius: var(--border-radius);
+    --bottom-right-radius: var(--border-radius);
+  }
+
+  .product--stacked .grid--collapsed-y .grid__item:last-child:nth-child(odd) {
+    --bottom-right-radius: var(--border-radius);
+  }
+
+  .product--stacked .grid--collapsed-y .grid__item:nth-last-child(2):nth-child(even):not(.product__media-item--full) {
+    --bottom-left-radius: var(--border-radius);
+  }
+
+  .product--stacked .grid--collapsed-x:not(.grid--collapsed-y) .grid__item:first-child,
+  .product--stacked .grid--collapsed-x:not(.grid--collapsed-y) .grid__item:nth-child(even) {
+    --top-left-radius: var(--border-radius);
+    --bottom-left-radius: var(--border-radius);
+  }
+
+  .product--stacked .grid--collapsed-x:not(.grid--collapsed-y) .grid__item:nth-child(odd),
+  .product--stacked .grid--collapsed-x:not(.grid--collapsed-y) .grid__item:last-child:nth-child(even) {
+    --top-right-radius: var(--border-radius);
+    --bottom-right-radius: var(--border-radius);
+  }
+}
```

Comment:
Here when there is no vertical space but some horizontal one, we're losing the border radius: 
![](https://screenshot.click/22-39-319no-dgjab.png)

Reply:
Should be fixed along with the other sliders